crypto/conf/conf_api.c

```diff
@@ -15,10 +15,16 @@
 #include <string.h>
 #include <openssl/conf.h>
 #include <openssl/conf_api.h>
+#include "conf_local.h"
 
 static void value_free_hash(const CONF_VALUE *a, LHASH_OF(CONF_VALUE) *conf);
 static void value_free_stack_doall(CONF_VALUE *a);
 
+OSSL_LIB_CTX *NCONF_get1_libctx(CONF *conf)
```

Comment:
This is wrong name! It should be `NCONF_get0_libctx`.

Reply:
updated commit push rename get1 to get0.